bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/slider/res/css/bootstrap-slider.css

```diff
@@ -0,0 +1,255 @@
+/*! =======================================================
+                      VERSION  5.3.7              
```

Comment:
I've noticed that v.6.x is released two days ago.
Do you want the newest version or 5.3.7 is good enough?